community/modules/scheduler/schedmd-slurm-gcp-v5-controller/README.md

```diff
@@ -0,0 +1,141 @@
+## Description
+
+This module creates a slurm controller node via the [SchedMD/slurm-gcp]
+[slurm\_controller\_instance] and [slurm\_instance\_template] modules.
+
+More information about Slurm On GCP can be found at the
+[project's GitHub page][SchedMD/slurm-gcp] and in the
+[Slurm on Google Cloud User Guide][slurm-ug].
+
+The [user guide][slurm-ug] provides detailed instructions on customizing and
+enhancing the Slurm on GCP cluster as well as recommendations on configuring the
+controller for optimal performance at different scales.
+
+[SchedMD/slurm-gcp]: https://github.com/SchedMD/slurm-gcp/tree/v5.0.2
+[slurm\_controller\_instance]: https://github.com/SchedMD/slurm-gcp/tree/v5.0.2/terraform/slurm_cluster/modules/slurm_controller_instance
+[slurm\_instance\_template]: https://github.com/SchedMD/slurm-gcp/tree/v5.0.2/terraform/slurm_cluster/modules/slurm_instance_template
+[slurm-ug]: https://goo.gle/slurm-gcp-user-guide.
+
+### Example
+
+```yaml
+- source: community/modules/scheduler/schedmd-slurm-gcp-v5-controller
+  kind: terraform
+  id: slurm_controller
+  use:
+  - network1
+  - homefs
+  - compute_partition
+```
+
+This creates a controller node connected to the primary subnetwork of
+`network1`. The controller will also have the `homefs` file system
+mounted and manage one partition, both declared in the `use`
+field. For more context see the
+[hpc-cluster-slurm-vt example](../../../examples/hpc-cluster-slurm-v5.yaml).
```

Comment:
This doesn't yet exist in this PR and just want to note that as an issue to track if the filename changes. I'm not really sure how I feel about an example "snippet" w/o any settings at all but I'll give it thought.

Reply:
Removed, and added a note to add it back in when I submit the example PR. Thanks for catching this.

Also, I added a setting for machine_type. Nothing is required if the network and partition are provided via `use`, but it doesn't hurt to show how it can be customized otherwise.